capi/src/visitors/image-references.ts

```diff
@@ -1,6 +1,6 @@
 import * as t from "../types";
 
-const IS_URL_REGEX = /^https?:\/\//i;
+const IS_ABSOLUTE_REGEX = /^https?:\/\//i;
```

Comment:
Can we name this `IS_URL_ABSOLUTE_REGEX`

Reply:
Done